### PR TITLE
Added Lead, Plastic, and Aluminium as new materials for the Autolathe

### PR DESCRIPTION
**Pull request by Generalcamo**
Inspired by Baystation, but sprites are my own.

This adds Lead, Plastic, and Aluminium as new materials for the autolathe. Several devices now use either Aluminium or Plastic instead of steel, depending on their sprite. Lead is slated for bullets, but Galena is added immediately for preparation for this.

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

File: code/game/machinery/autolathe/autolathe_datums.dm [hunk 1]
```diff
@@ -115,6 +115,13 @@
 	category = "General"
 	is_stack = 1
 
+/datum/autolathe/recipe/glass
```
Comment (NonQueueingMatt) [inline code/game/machinery/autolathe/autolathe_datums.dm:118]: ?

File: code/game/machinery/autolathe/autolathe.dm [hunk 2]
```diff
@@ -290,6 +290,9 @@
 
 	storage_capacity[DEFAULT_WALL_MATERIAL] = mb_rating * 25000
 	storage_capacity[MATERIAL_GLASS] = mb_rating * 12500
+	storage_capacity[MATERIAL_ALUMINIUM] = mb_rating * 25000
+	storage_capacity[MATERIAL_PLASTIC] = mb_rating * 12500
+	storage_capacity[MATERIAL_LEAD] = mb_rating * 12500
```
Comment (NonQueueingMatt) [inline code/game/machinery/autolathe/autolathe.dm:295]: Did you test that this works as in game without editing the TGUI?

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/machinery/autolathe/autolathe_datums.dm (hunk 1 above)
Comment (Generalcamo) [reply code/game/machinery/autolathe/autolathe_datums.dm:118]: Merge error, removed

File: code/game/machinery/autolathe/autolathe.dm (hunk 2 above)
Comment (Generalcamo) [reply code/game/machinery/autolathe/autolathe.dm:295]: Can confirm it does:
image

**Review by NonQueueingMatt — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**